### ISSUE TITLE
Download unverified track - message

### ISSUE THREAD

**Issue by Tengu-san** [nukeop/nuclear#1513]
Hello,
I would like to suggest to add an error message, when one try to download an unverified track.
Saying that **only verified track can be download**.
This will make the experience more clearer.
Thanks a lot for the good job

**Comment by nukeop** [comment]
That's not true, unverified tracks download as normal. They're just not guaranteed to be always correct. Is that what you meant?

**Comment by Tengu-san** [comment]
In my case, unverified tracks never download
